.eslintrc

```diff
@@ -17,45 +17,51 @@
     "commonjs": true,
     "es6": true,
     "mocha": true,
-    "jest": true,
+    "jest": true
   },
   "globals": {
     "PerformanceObserver": true,
     "__BUILDTYPE__": true,
     "__API__": true,
-    "__MEGAMENU_CONFIG__": true,
+    "__MEGAMENU_CONFIG__": true
   },
   "rules": {
-    // Override airbnb style.
-    "camelcase": [2, { "properties": "always" }],
-    "va-enzyme/unmount": 2,
-    "fp/no-proxy": 2, // IE 11 has not polyfill for Proxy 
+    /* || Eslint main rules || */
+    "camelcase": [2, { "properties": "always" }], // Override airbnb style.
     "func-names": 2,
-    "import/no-extraneous-dependencies": 0,
     "no-console": 2,
-    "no-negated-condition": 0, // Sometimes negated conditions are easier to understand.
-    "no-plusplus": 0,
-    "no-underscore-dangle": 0, // We have build flags that use this
     "no-unused-vars": [
       2,
       { "args": "after-used", "argsIgnorePattern": "^_", "vars": "local" }
     ],
     "no-restricted-imports": ["error", "raven"],
     "prefer-rest-params": 2,
-    "quote-props": [2, "as-needed", { "keywords": false }],
 
+    /* || va-enzyme custom plugin || */
+    "va-enzyme/unmount": 2,
+
+    /* || fp plugin || */
+    "fp/no-proxy": 2, // IE 11 has not polyfill for Proxy
```

Comment:
Typo: `has not polyfill` => `has no polyfill`.

Reply:
```suggestion
    "fp/no-proxy": 2, // IE 11 has no polyfill for Proxy
```